app/Http/Controllers/BlogController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Blog;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class BlogController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+        $blogs = Blog::all();
+        return view('blog/index', ['blogs'=>$blogs]);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        //
+        $request->validate([
+            'title' => 'required',
+            'content' => 'required',
+        ]);
+        $blog = new Blog;
+        $blog->user_id = Auth::user()->id;
+        $blog->title = $request->title;
+        $blog->content = $request->content;
+        $blog->save();
+        return redirect()->back();
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
```

Comment:
If not have content yet , please remove function

Reply:
i fixed